test/custom_validation_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+describe Lotus::Validations, 'custom validations' do
+  describe '#validate keyword' do
+    it "can mimic presence validator" do
+      validator = CustomPresenceValidation.new(name: '')
+      validator.valid?.must_equal false
+      validator.name = "X"
+      validator.valid?.must_equal true
+    end
+  end
+
+  describe '#validate keyword' do
+    it "can mimic presence validator" do
```

Comment:
One test is for `CustomPresenceValidation` and another for `CustomAttributesValidation`. I'll do a general cleanup & maybe some more testing soon!